organization/client.go

```diff
@@ -117,6 +120,65 @@ func (c *Client) Delete(ctx context.Context, id string) (*clerk.DeletedResource,
 	return organization, err
 }
 
+type UpdateLogoParams struct {
+	clerk.APIParams
+	File           multipart.File `json:"-"`
+	UploaderUserID *string        `json:"-"`
+	Filename       *string        `json:"-"`
```

Comment:
❓ Is the filename parameter useful? Do we use it for mimetype detection?

Reply:
Looks like it's not useful at all. Removed. :smile: